platform/commonUI/browse/src/windowing/NewTabAction.js

```diff
@@ -38,6 +38,7 @@ define(
 
             this.urlService = urlService;
             this.open = function () {
+                arguments[0] += "&tabOpened=true"; // add tabOpened param
```

Comment:
instead of `tabOpened`, use `hideInspector` and `hideTree`.  Don't need the complexity of all three.

This comment is literally duplicating what the code does, so unnecessary.  Let code speak for itself.